sdk/monitor/Azure.Monitor.Query/tests/MonitorQueryTestEnvironment.cs

```diff
@@ -25,5 +26,36 @@ public class MonitorQueryTestEnvironment : TestEnvironment
         public string ConnectionString => GetRecordedOptionalVariable("CONNECTION_STRING");
         public string StorageAccountId => GetRecordedOptionalVariable("STORAGE_ID");
         public string StorageAccountConnectionString => GetRecordedOptionalVariable("STORAGE_CONNECTION_STRING");
+
+        public string GetAudience()
+        {
+            Uri authorityHost = new(AuthorityHostUrl);
+
+            if (authorityHost == AzureAuthorityHosts.AzurePublicCloud)
+            {
+                if (AzureAuthorityHosts.AzurePublicCloud.ToString() == LogsQueryAudience.AzurePublicCloud)
```

Comment:
I'm a little confused as to why we compare these two AzurePublicCloud definitions. Shouldn't we just compare the authority host against LogsQueryAudience directly?

Reply:
Yes - this implementation was wrong